source/includes/examples/_cart-pole.html.md

```diff
@@ -35,7 +35,8 @@ The schema `Action` names a record — `action` —  and assigns it a constraine
 
 ```inkling
 schema CartPoleConfig
-  Int8 episode_length,
+  # Note: this is a signed Int8 because -1 is used for "run until pole drops"
```

Comment:
Makes sense, though I think there should be some comments in the code examples -- it took me a while to figure out whether comments were even allowed, and what the syntax was. Still not sure if there's a multi-line comment syntax.